src/test/java/com/google/api/codegen/testdata/discogapic/java/java_fakebilling.v1.json.baseline

```diff
@@ -548,6 +553,16 @@ public final class BillingAccountsGetHttpRequest {
       return this;
     }
 
+    public List<String> getFooRepeated() {
+      return fooRepeated;
+    }
+
+    public Builder setFooRepeated(List<String> fooRepeated) {
+      this.fooRepeated = new LinkedList<>();
```

Comment:
Ahh! Here is an important clarification: Guava shouldn't be used on the surface (i.e. you shouldn't see any Guava types when you look at the javadoc for our library), but it can be used in the implementation.